### ISSUE TITLE
Support '0' as a valid placeholder value in Plurr preview

### ISSUE THREAD

**Issue by iafan** [evernote/zing#467]
For strings that contain Plurr placeholders, we render a preview panel with the ability to enter the number and see the rendered formatted string. However, placing `0` there makes Zing think that no number is present, and the preview is not rendered:

![Screen Shot 2020-04-24 at 12 48 50](https://user-images.githubusercontent.com/1728158/80251149-fd51d080-8629-11ea-86d6-ed2bb63d1ef6.png)

Suggestion: use a strict comparison with an empty string `''` to determine if the field is not empty.

**Comment by benjamimo1** [comment]
@iafan By plum we mean this: https://github.com/loctools/plurr ?

**Comment by iafan** [comment]
@benjamimo1 correct!

**Comment by benjamimo1** [comment]
any special behavior for 0 as a number that you would like to see implemented?

**Comment by benjamimo1** [comment]
If we input all necessary strings:
<img width="890" alt="Screen Shot 2020-05-20 at 10 20 08" src="https://user-images.githubusercontent.com/26126311/82457565-90dcbc80-9a83-11ea-9a1e-abb8a9894806.png">

If we don't:
<img width="897" alt="Screen Shot 2020-05-20 at 10 20 13" src="https://user-images.githubusercontent.com/26126311/82457589-989c6100-9a83-11ea-95e6-13d1fb08154f.png">

Is "Empty string rendered" a different case scenario as "Please enter placeholder"?
Are we aiming for two different message for each scenario? I don't understand how they are different.
Or should I just change the message altogether without a change in logic?

**Comment by iafan** [comment]
Yes, these are different cases. The first one is when you have missing (empty) placeholder values. The second one is an empty string as returned from Plurr formatting function.

In your screenshot above, you will never get an empty string in the output, regardless of the parameters. But there can be units which may produce an empty string as an output. [See this example](http://iafan.github.io/plurr-demo/#locale=en&autoPlurals=true&strict=true&s=%7BN%3Azero%7C%7Ctwo+and+mode%7D&p%5BN%5D=1) (try to change N to `0`, `1`, and `2`)

**Comment by benjamimo1** [comment]
OK, will give it a look now, assuming two different cases.
